### ISSUE TITLE
Generating WeKan OpenAPI docs does not work

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#4917]
@bentiss

Hi ! Do you have time to fix generating WeKan OpenAPI docs? I'm using Ubuntu 23.04 amd64 with XFCE desktop.

I tried to fix it at https://github.com/wekan/wekan/commit/97c6b6531502d09cbd6c267212651f3036338a11

But I still get this error message:

```
✓ Read source file!
Failed to parse the source OpenAPI document
```

**Comment by xet7** [comment]
I tried to build docs this way:
```
git clone https://github.com/wekan/wekan

cd wekan

./releases/rebuild-docs.sh 6.87
```

**Comment by xet7** [comment]
WeKan v6.87 will have some new API https://github.com/wekan/wekan/pull/4911 , but I'm not able to build docs for it.

**Comment by bentiss** [comment]
Hi!

I think it's should be fixed in #4918 (at least the yaml file can be ingested by the public ReDoc test server).

**Comment by xet7** [comment]
Thanks a lot !!
